plotting/tests/test_plot_handler.py

```diff
@@ -0,0 +1,204 @@
+        When: Instrument name is WISH and no plot type is passed to the class
+        """
+        expected = "WISH4321*.jpg"
+        actual = self.test_wish_none_plottype._regexp_for_file_name(self.test_wish_none_plottype.plot_type[1])
+        self.assertEqual(expected, actual)
+        expected = "WISH4321*.tiff"
+        actual = self.test_wish_none_plottype._regexp_for_file_name(self.test_wish_none_plottype.plot_type[-1])
+        self.assertEqual(expected, actual)
+
+    @patch('utils.clients.sftp_client.SFTPClient.__init__', return_value=None)
+    @patch('utils.clients.sftp_client.SFTPClient.get_filenames')
+    def test_check_for_plot_files_mari_png_plottype(self, mock_get_filenames, mock_client_init):
+        """
+        Test: sftpclient.get_filenames is called with the correct parameters if only one plot_type is used
+        When: sftpclient.get_filenames is used to look for existing plot files
+        :return:
+        """
+        self.test_mari_png_plottype._check_for_plot_files()
+        # client = SFTPClient()
+        mock_client_init.assert_called_once()
+        mock_get_filenames.assert_called_once_with(
+            server_dir_path="/instrument/MARI/RBNumber/RB12345678/1234/autoreduced/", regex="MAR[I]1234*.png")
+
+    @patch('utils.clients.sftp_client.SFTPClient.__init__', return_value=None)
+    @patch('utils.clients.sftp_client.SFTPClient.get_filenames')
+    def test_check_for_plot_files_wish_none_plottype(self, mock_get_filenames, mock_client_init):
+        """
+        Test: sftpclient.get_filenames is called with the correct parameters when multiple plot_types are used
+        When: sftpclient.get_filenames is used to look for existing plot files
+        :return:
+        """
+        self.test_wish_none_plottype._check_for_plot_files()
+        mock_client_init.assert_called_once()
+        # check number of times the sftpclient.get_filenames method was called
+        expected = 5
+        actual = mock_get_filenames.call_count
+        self.assertEqual(expected, actual)
+        # check that the sftpclient.get_filenames method was called with the right parameters
+        mock_get_filenames.assert_any_call(server_dir_path="/instrument/WISH/RBNumber/RB87654321/4321/autoreduced/",
+                                           regex="WISH4321*.png")
+        mock_get_filenames.assert_any_call(server_dir_path="/instrument/WISH/RBNumber/RB87654321/4321/autoreduced/",
+                                           regex="WISH4321*.jpg")
+        mock_get_filenames.assert_any_call(server_dir_path="/instrument/WISH/RBNumber/RB87654321/4321/autoreduced/",
+                                           regex="WISH4321*.bmp")
+        mock_get_filenames.assert_any_call(server_dir_path="/instrument/WISH/RBNumber/RB87654321/4321/autoreduced/",
+                                           regex="WISH4321*.gif")
+        mock_get_filenames.assert_any_call(server_dir_path="/instrument/WISH/RBNumber/RB87654321/4321/autoreduced/",
```

Comment:
Rather than having this string copied and pasted each time, I would create a variable within `setUp()` for this string, and use that variable name throughout

Reply:
I would do this for all constant values used across the test file.
See the `setUp()` method of `utils/clients/tests/test_sftp_client.py` for an example